### ISSUE TITLE
Error while parsing values that start with certain characters

### ISSUE THREAD

**Issue by myuce** [Perryvw/PHPValveKV#2]
I've been trying to parse some vmt files and realized it throws an exception when a file has kvps which don't have quotes. I thought it was because they didn't have quotes but when I check other files, I had realized that some of them start with characters like dots and brackets. An example which makes the parser throw an error can be seen below.

```
"LightmappedGeneric"
{
	"$basetexture" "Brick/brickfloor001a"
	"$surfaceprop" "brick"

	"$detail" "detail\noise_detail_01"
	"$detailscale" "7.740"
	"$detailblendfactor" ".8"

	"$detailblendmode" 0
	"%keywords" "c17downtown"
}
```

**Comment by Perryvw** [comment]
Looks to be an easy fix, will implement later today.

**Comment by Perryvw** [comment]
Fixed with f83529d63152816ab46cb941a569a80f012950c1

**Comment by myuce** [comment]
I also get the same exception while parsing the string below. It's probably because of the bracket at the beginning of the value.

```
"VertexLitGeneric"
{
	"$basetexture" "Models/props_interiors/LightBulb01a"
	"$envmap" "env_cubemap"
	"$envmapmask" "Models/props_interiors/LightBulb01a_mask"
	"$envmaptint" "[ .4 .4 .4]"
	"$envmapcontrast" "1"
	"$envmapsaturation" [.5 .5 .5]
	"$surfaceprop" "glass"
	"$selfillum" 1	
	
}
```

**Comment by Perryvw** [comment]
Fixed that in acff815d785ad91af4557baaf12f8a95a936cb59
